src/astro/files/base.py

```diff
@@ -98,6 +102,7 @@ def get_files(
     conn_id: Optional[str] = None,
     filetype: Union[FileType, None] = None,
     normalize_config: Optional[dict] = None,
+    if_file_doesnt_exist: FileNotFoundStrategy = "exception",
```

Comment:
Can you think of a use-case where someone would want to suppress the error? It most certainly should be error if the file doesn't exists, as the operator is no-op

Let's simplify this an just error when the file doesn't exist

Reply:
@kaxil - The initial intent was to cater to cases where we have dags that monitor a folder for the presence of files and process them if they exist. But I think we can achieve that with the file_sensor operator as well.

Reply:
Yeah, that should not be the use case of this operator. We have an async version of FileSensor for that too.